packages/app-desktop/ElectronAppWrapper.ts

```diff
@@ -355,6 +368,17 @@ export default class ElectronAppWrapper {
 		this.electronApp_.on('activate', () => {
 			this.win_.show();
 		});
+
+		this.electronApp_.on('open-url', (_event: any, url: string) => {
```

Comment:
Should "event" because the underscore is only when the variable is unused (to disable TypeScript type checking)

Reply:
done